deploy/playbooks/roles/repos/templates/distributions

```diff
@@ -0,0 +1,87 @@
+Codename: sid
+Suite: stable
+Components: main
+Architectures: amd64 armhf i386 source
+Origin: Inktank
+Description: Ceph distributed file system
+DebIndices: Packages Release . .gz .bz2
+DscIndices: Sources Release .gz .bz2
+Contents: .gz .bz2
+SignWith: 17ED316D
+
+Codename: wheezy
```

Comment:
also, since this is a template, maybe we should make use of the helpers? e.g. make it a loop? hopefully that doesn't mean too much work, maybe we can revisit if so